editor/property_editor.h

```diff
@@ -1,4 +1,4 @@
-﻿/*************************************************************************/
+/*************************************************************************/
```

Comment:
Your IDE added a BOM here, could you amend the commit to remove it?

Reply:
Can you please manually fix it? Can't see a BOM, nor in Visual Studio nor in Notepad++ nor in AkelPad. Shows me typical UTF-8 without any bom.

Reply:
And clang-format also doesn't fix anything.

Reply:
Nevermind, it looks like you *removed* an existing BOM, so that's good.